### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by Qarola**
✔️ Created a skeleton to FE.
✔️  Added dependecies.

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "client",
```
Comment (gpgpublickey) [inline client/package.json]: Please update the name to Mapanimal @Qarola

File: client/public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```
Comment (gpgpublickey) [inline client/public/manifest.json:2]: Please replace the name to Mapanimal

File: client/src/App.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        <p>
+          Edit <code>src/App.js</code> and save to reload.
+        </p>
+        <a
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn React
```
Comment (gpgpublickey) [inline client/src/App.js:18]: Please create a task under the same story to make an "under construction landing page"

File: client/src/index.css [hunk 4]
```diff
@@ -0,0 +1,13 @@
+body {
```
Comment (gpgpublickey) [inline client/src/index.css]: please move this file to a folder for styles

File: client/src/redux/actions/index.js [hunk 5]
```diff
@@ -0,0 +1,2 @@
+//import axios from "axios";
```
Comment (gpgpublickey) [inline client/src/redux/actions/index.js:1]: Please remove it if it's not used, for now, leave commented code is a code smell.

File: client/src/index.css (hunk 4 above)
Comment (gpgpublickey) [reply client/src/index.css]: consider translate to sass

**Review by gpgpublickey — CHANGES_REQUESTED** [review]

File: client/src/redux/actions/index.js (hunk 5 above)
Comment (Qarola) [reply client/src/redux/actions/index.js:1]: If I remove it, the folder does not upload. I uploaded it to have some structure of all the folders to use. They may not all be used in sprint 0, but they are there.

Comment (gpgpublickey) [reply client/src/redux/actions/index.js:1]: Ok @Qarola thx for the feedback

**Review by gpgpublickey — APPROVED** [review]

**Comment by gpgpublickey** [comment]
@Qarola approved, please request another review in telegram and ship it! (merge it)

**Review by gpgpublickey — APPROVED** [review]

**Comment by gpgpublickey** [comment]
@dnlaMrno @JoanaRomay @Floriana19 @javierpz96   please approve if pass your review.

**Pull request merged**